tests/services/test_availability_service.py

```
@@ -3,6 +3,8 @@
 from datetime import datetime, timedelta
 from app.services.availability_service import AvailabilityService
 from app.models.dto import AvailabilityRequest, RoomAvailability, RoomDetail
+from datetime import datetime, date, timedelta
```

Ruff (0.15.4)

[error] Redefinition of unused `datetime` from line 3: `datetime` redefined here

Remove definition: `datetime`

(F811)